batchflow/models/torch/base.py

```diff
@@ -122,18 +120,19 @@ class TorchModel:
         Examples:
 
         - ``{'optimizer': 'Adam'}``
-        - ``{'optimizer': ('SparseAdam', {'lr': 0.01})}``
-        - ``{'optimizer': {'name': 'Adagrad', 'initial_accumulator_value': 0.01}``
-        - ``{'optimizer': {'name': MyCustomOptimizer, momentum=0.95}}``
+        - ``{'optimizer': {'name': 'SparseAdam', 'lr': 0.01}}``
+        - ``{'optimizer': {'name': 'Adagrad', 'initial_accumulator_value': 0.01}}``
+        - ``{'optimizer': {'name': MyCustomOptimizer, 'momentum': 0.95}}``
 
-    decay : str, tuple, dict
+    decay : dict, list
         Learning rate decay algorithm, might be defined in multiple formats.
-        All decays require to have ``n_iters`` as a key in a configuration
+        All decays require to have 'frequency' as a key in a configuration
         dictionary that contains the number of iterations in one epoch.
+        Each decay might have optional parameters 'first_iter' and 'last_iter'
+        that defines iterations between which decay works.
```

Comment:
`range of iterations where decay is at work`?
something like that, not `between`